src/main/java/uk/gov/hmcts/reform/juddata/camel/route/LeafTableRoute.java

```diff
@@ -77,20 +113,42 @@ public void startRoute() throws FailedToCreateRouteException {
                         @Override
                         public void configure() throws Exception {
 
+
+                            onException(RouteFailedException.class, ValidationException.class, RuntimeException.class)
+                                    .handled(true)
+                                    .process(failureProcessor)
+                                    .process(schedulerAuditProcessor)
+                                    .markRollbackOnly()
+                                    .end();
+
                             //logging exception in global exception handler
                             onException(Exception.class)
                                     .handled(true)
-                                    .process(exceptionProcessor);
+                                    .process(exceptionProcessor)
+                                    .end()
+                                    .process(schedulerAuditProcessor);
 
                             String[] directRouteNameList = createDirectRoutesForMulticast(leafRoutesList);
+                            //add last child route as  archival
+                            directRouteNameList = copyOf(directRouteNameList, directRouteNameList.length + 1);
+                            directRouteNameList[directRouteNameList.length - 1] = leafArchivalRoute;
 
                             //Started direct route with multicast all the configured routes eg.application-jrd-leaf-router.yaml
                             //with Transaction propagation required
                             from(startLeafRoute)
                                     .transacted()
                                     .policy(springTransactionPolicy)
                                     .multicast()
-                                    .stopOnException().to(directRouteNameList).end();
+                                    .stopOnException().to(directRouteNameList).end()
+                                    .process(schedulerAuditProcessor); //To do replace with Processor
+
```

Comment:
when is this supposed to be replaced?

Reply:
i think it is replaced just need to remove comment.